airbyte-integrations/connectors/source-redshift/src/main/resources/spec.json

```
@@ -19,24 +20,40 @@
         "minimum": 0,
         "maximum": 65536,
         "default": 5439,
-        "examples": ["5439"]
+        "examples": ["5439"],
+        "order": 2
       },
       "database": {
         "title": "Database",
         "description": "Name of the database.",
         "type": "string",
-        "examples": ["master"]
+        "examples": ["master"],
+        "order": 3
+      },
+      "schemas": {
+        "title": "Schemas",
+        "description": "The list of schemas to sync from. Specify one or more explicitly or keep empty to process all schemas.Case sensitive.",
```

Nitpick. Missed a space here.

```suggestion
        "description": "The list of schemas to sync from. Specify one or more explicitly or keep empty to process all schemas. Schema names are case sensitive.",
```